packages/EasyErrorHandler/docs/config.md

```diff
@@ -47,13 +47,14 @@ Laravel has the following additional configuration option:
 
 Symfony has the following additional configuration options:
 
-| Configuration                      | Default    | Description                                                                                              |
```

Comment:
I see we don't have `errorCodesInterface` in the common configuration section above. Let's add it.

Reply:
Added. Please check it